tsdb/engine/tsm1/compact.go

```diff
@@ -306,7 +303,7 @@ func (c *Compactor) writeNewFiles(generation, sequence int, iter KeyIterator) ([
 	for {
 		sequence++
 		// New TSM files are written to a temp file and renamed when fully completed.
-		fileName := filepath.Join(c.Dir, fmt.Sprintf("%09d-%09d.%s.tmp", generation, sequence, "tsm1dev"))
+		fileName := filepath.Join(c.Dir, fmt.Sprintf("%09d-%09d.%s.tmp", generation, sequence, "tsm"))
```

Comment:
I changed to `tsm` on purpose. The version is now contained within the file itself so it doesn't make sense to have the `1` extension.